driverconfig/config.yaml

```diff
@@ -116,6 +116,7 @@ drivers:
       - configVersion: v2.6.0
         useDefaults: true
```

Comment:
@shefali-malhotra, please do the changes for K8s 1.21 to use the latest version of sidecars.

Reply:
@shefali-malhotra , i dont see these changes done